### ISSUE TITLE
Tests failling

### ISSUE THREAD

**Issue by pwener** [fga-gpp-mds/2017.1-Cadernos-API#11]
The `users_controller_spec.rb` tests are failling in my env. Someone know  what is happening? @fabio1079 @victorcmoura

**Comment by pwener** [comment]
Some cases:

```
 3) UsersController GET #index assigns all users as @users
     Failure/Error: user = User.create! valid_attributes
     
     ActiveRecord::RecordInvalid:
       Validation failed: Email has already been taken
     # ./spec/controllers/users_controller_spec.rb:49:in `block (3 levels) in <top (required)>'

  4) UsersController GET #show assigns the requested user as @user
     Failure/Error: user = User.create! valid_attributes
     
     ActiveRecord::RecordInvalid:
       Validation failed: Email has already been taken
     # ./spec/controllers/users_controller_spec.rb:57:in `block (3 levels) in <top (required)>'

  5) UsersController POST #create with valid params creates a new User
     Failure/Error:
       expect {
         post :create, params: {user: valid_attributes}, session: valid_session
       }.to change(User, :count).by(1)
     
       expected #count to have changed by 1, but was changed by 0
     # ./spec/controllers/users_controller_spec.rb:66:in `block (4 levels) in <top (required)>'

  6) UsersController POST #create with valid params assigns a newly created user as @user
     Failure/Error: expect(assigns(:user)).to be_persisted
       expected `#<User id: nil, name: "ValidName", email: "validEmail@mail.com", email_confirmation: nil, password_digest: "$2a$04$M907j/AUU.1YVHxKV50Ssuvn1IFBzFhFPZwqQQQnbCU...", created_at: nil, updated_at: nil>.persisted?` to return true, got false
     # ./spec/controllers/users_controller_spec.rb:74:in `block (4 levels) in <top (required)>'

  7) UsersController POST #create with valid params redirects to the created user
     Failure/Error: expect(response).to have_http_status(201)
       expected the response to have status code 201 but it was 422
     # ./spec/controllers/users_controller_spec.rb:79:in `block (4 levels) in <top (required)>'

  8) UsersController PUT #update with valid params updates the requested user
     Failure/Error: user = User.create! valid_attributes
     
     ActiveRecord::RecordInvalid:
       Validation failed: Email has already been taken
     # ./spec/controllers/users_controller_spec.rb:102:in `block (4 levels) in <top (required)>'

  9) UsersController PUT #update with valid params assigns the requested user as @user
     Failure/Error: user = User.create! valid_attributes
     
     ActiveRecord::RecordInvalid:
       Validation failed: Email has already been taken
     # ./spec/controllers/users_controller_spec.rb:109:in `block (4 levels) in <top (required)>'

  10) UsersController PUT #update with valid params return the user once it is created
      Failure/Error: user = User.create! valid_attributes
      
      ActiveRecord::RecordInvalid:
        Validation failed: Email has already been taken
      # ./spec/controllers/users_controller_spec.rb:115:in `block (4 levels) in <top (required)>'

  11) UsersController PUT #update with invalid params assigns the user as @user
      Failure/Error: user = User.create! valid_attributes
      
      ActiveRecord::RecordInvalid:
        Validation failed: Email has already been taken
      # ./spec/controllers/users_controller_spec.rb:123:in `block (4 levels) in <top (required)>'

  12) UsersController DELETE #destroy destroys the requested user
      Failure/Error: user = User.create! valid_attributes
      
      ActiveRecord::RecordInvalid:
        Validation failed: Email has already been taken
      # ./spec/controllers/users_controller_spec.rb:132:in `block (3 levels) in <top (required)>'

  13) UsersController DELETE #destroy redirects to the users list
      Failure/Error: user = User.create! valid_attributes
      
      ActiveRecord::RecordInvalid:
        Validation failed: Email has already been taken
      # ./spec/controllers/users_controller_spec.rb:139:in `block (3 levels) in <top (required)>'
```

**Comment by pwener** [comment]
Fixed in #12
